### PR TITLE
Default to a minimum height of 100 for stories. Allow override.

### PR DESCRIPTION
**Pull request by timhaines**

### CODE REVIEW

**Comment by timhaines** [comment]
I expect the integration tests (Percy Snapshots) to fail until we've shipped the updated percy-js and deploy the api updates.

**Comment by timhaines** [comment]
@fotinakis Have confirmed this locally against the minimum_height branches of everything, and confirmed it works.  

After the new release of percy-client (percy-js), and once the API has been shipped, the Percy Storybook screenshots will have a default height of 100.

File: packages/react-percy-storybook/src/__tests__/uploadStories-tests.js [hunk 1]
```diff
@@ -13,9 +13,9 @@ it('uploads stories for each test case', async () => {
     ];
     const storyHtml = '<html></html>';
 
-    await uploadStories(percyClient, build, stories, [328, 768], assets, storyHtml);
+    await uploadStories(percyClient, build, stories, [328, 768], 100, assets, storyHtml);
```
Comment (fotinakis) [inline packages/react-percy-storybook/src/__tests__/uploadStories-tests.js]: Man I hate positional arguments so much. :| ;) Ignoring for now.

Comment (timhaines) [inline packages/react-percy-storybook/src/__tests__/uploadStories-tests.js]: lol

**Pull request merged**